src/parsers/in.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import P from 'parsimmon'
+import crap from './crap'
+
+import type { TupleNodeType, PrimitiveNodeType } from '../types'
+
+import StringParser from './primitives/string'
+import ObjectParser from './object'
+
+export default P.seq(
+  StringParser.trim(crap),
```

Comment:
This only works for string literals. It'd be much useful if the operator worked with any kind of value, some examples:

```
$myString in {} where $mystring = "Hello"
```

or 

```
["Hello", "asd"][0] in {}
```